src/prometheus_fastapi_instrumentator/middleware.py

```diff
@@ -178,12 +178,14 @@ async def send_wrapper(event: ASGISendEvent) -> None:
                 )
 
                 for instrumentation in self.instrumentations:
-                    instrumentation(info)
+                    if instrumentation:
+                        instrumentation(info)
 
                 await asyncio.gather(
                     *[
                         instrumentation(info)
                         for instrumentation in self.async_instrumentations
+                        if instrumentation
```

Comment:
The instrumentation functions can now be `None`. Happens when middleware gets applied again and the respective function in `metrics` returns `None` instead of instrumentation function due to try catch.